core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts

```diff
@@ -183,7 +183,7 @@ export class DotContentEditorComponent implements OnInit, OnChanges {
         if (this.getcontainerStructures.length === 0) {
             this.getcontainerStructures.push(
                 new FormGroup({
-                    code: new FormControl('', [Validators.required]),
+                    code: new FormControl(''),
```

Comment:
Can we test this to make sure no one marks it as `required`?

Reply:
Noted

Reply:
Updated